shared/chat/conversation/messages/react-button/emoji-row/data.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import {memoize} from '../../../../../util/memoize'
+import {isMobile} from '../../../../../constants/platform'
+
+const getFrequently = memoize<void, void, void, void, {get: number => Array<string>}>(
+  () => require('emoji-mart').frequently
```

Comment:
This means we rely on some facility of emoji-mart to say which ones are frequent>

Reply:
Yeah it does. It uses `LocalStorage` which we can do when we transition away from emoji-mart